router/image/templates/ssl.conf

```diff
@@ -0,0 +1,23 @@
+# set the allowed protocols
+ssl_protocols {{ or (getv "/deis/router/sslProtocols") "TLSv1 TLSv1.1 TLSv1.2" }};
+
+# turn on session caching to drastically improve performance
+ssl_session_cache {{ or (getv "/deis/router/sslSessionCache") "shared:SSL:32m" }};
```

Comment:
this session is only shared amongst all workers across a single router. Doesn't that defeat the purpose when the next incoming request is redirected to another router? The session would be lost.